src/Formatter/AbstractFormatter.php

```diff
@@ -48,33 +34,4 @@ public static function accepts()
      * @return string
      */
     abstract protected function body(PayloadInterface $payload);
-
-    /**
-     * Get the response status from the payload.
-     *
-     * @param PayloadInterface $payload
-     *
-     * @return integer
-     */
-    public function status(PayloadInterface $payload)
```

Comment:
@AlexMasterov payload cannot be modified in responders because the payload is immutable. Modifications would affect the current responder only.